meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx

```diff
@@ -2,14 +2,49 @@ import React, { useEffect, useState } from 'react'
 import django from 'django'
 import { BudgetingProposalListItem } from './BudgetingProposalListItem'
 import { Pagination } from './Pagination'
+import { FilterBar } from './FilterBar'
+
+// filter mock, naive assumption of object structure.
+// to be replaced and adjusted, depending on what comes
+// from backend at the end
+const FILTERS = [
```

Comment:
@fuzzylogic2000 currently the frontend works with these object structure assumption. not sure if it makes sense, not sure if it is possible to pass the query name (here `type`) from backend?

Reply:
I guess that is a question for @Rineee as she works on the category and ordering choices now. Should filter name and type also come from the backend? Or do we assume we know the backend and just add the choices?

Reply:
it would be nice if we can get these infos (type, name) from backend. so we have a single source of truth.
i was wondering if the `All` state inside of choices should come from backend too? if so i can just fetch everything (except position) from backend and do not need to "serialize" in frontend again. but not sure if this is the responsibility of our backend, though. what do you think?

Reply:
Well, I love when the backend does all the logic. :)

Reply:
Hahha, me too :) Will look into it now and see how I we can provide you all the info best!